network/p2p/peer_tracker.go

```diff
@@ -133,10 +131,11 @@ func (p *peerTracker) shouldTrackNewPeer() bool {
 	// 5000               | 7.124576406741286e-218
 	//
 	// In other words, the probability drops off extremely quickly.
-	newPeerProbability := math.Exp(-float64(numResponsivePeers) * newPeerConnectFactor)
+	newPeerProbability := stdmath.Exp(-float64(numResponsivePeers) * newPeerConnectFactor)
 	return rand.Float64() < newPeerProbability // #nosec G404
 }
 
+// TODO get rid of minVersion
```

Comment:
as far as i can tell `minVersion` is no longer needed because all nodes support state sync now

Reply:
Only for C-chain. I'd keep it in anticipation of P-chain and X-chain state sync